leptos-spin-macro/src/lib.rs

```diff
@@ -17,3 +17,4 @@ pub fn server(args: proc_macro::TokenStream, s: TokenStream) -> TokenStream {
         Ok(s) => s.to_token_stream().into(),
     }
 }
+/Users/sam/Projects/start-spin/src
```

Comment:
I think this is a rogue paste

Reply:
Yeah, I thought so too. I need to remove that, they didn't get to it

Reply:
Hi Sorry, I've been out of the loop on this one. Yes rogue paste.